src/components/App.js

```diff
@@ -0,0 +1,26 @@
+import '../App.css';
+import { Route, Routes } from 'react-router-dom/dist';
+import Missions from './Missions';
+import MyProfile from './MyProfile';
+import Rockets from './Rockets';
+import Navigation from './Navigation';
+
+function App() {
+  return (
+    <div className="App">
+      <div>
+        <Navigation />
+        <Routes>
```

Comment:
- [ ] `<NavLink` /> should be used for navigation links. Please kindly fix that.

Reply:
@emmanuellekamwa Thanks a lot for your review!